tools/generator/templates/champions/cxx/interface_cxx.cc.jinja2

```diff
@@ -0,0 +1,34 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+// Copyright (c) 2020 Association Prologin <association@prologin.org>
+
+// This file is generated. It contains the code to call the API functions from
+// the C++ language. Do not edit this file.
+
+#include "api.hh"
+
+// This file only contains simple C++ wrappers arround the API, that are
+// basically already C++ functions, but with an `extern "C"`.
+{% for func in game.function %}
+extern "C" {{ func.fct_ret_type|cxx_type }} api_{{ func.fct_name }}({{ func.fct_arg|cxx_args }});
```

Comment:
func|cxx_prototype, but we need to add a prefix= option to that filter to handle `api_`

Reply:
ok